css/css-grid/alignment/references/grid-baseline-justify-001-ref.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<title>
+	Grid Item (First) Baseline Inline-Axis Alignment: auto-width grid, auto column, auto items
+</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#column-align">
+<link rel="help" href="https://www.w3.org/TR/css-grid-1/#algo-content">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#baseline-values">
+<link rel="match" href="references/grid-baseline-align-001-ref.html">
```

Comment:
This is also a failing lint. This isn't a self-reference, but probably also copy-pasted bits that should be deleted?